packages/block-library/src/missing/edit.native.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * External dependencies
+ */
+import { View, Text } from 'react-native';
+
+/**
+ * WordPress dependencies
+ */
+import { Icon } from '@wordpress/components';
+import { coreBlocks } from '@wordpress/block-library';
+import { normalizeIconObject } from '@wordpress/blocks';
+import { Component } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import styles from './style.scss';
+
+export default class UnsupportedBlockEdit extends Component {
+	render() {
```

Comment:
This could be written as a stateless component, since we are not using local state or lifecycle methods.
At my understanding, they are quite faster.